File: src/test/java/com/technews/scheduler/ReleasesJavaSchedulerTest.java
package com.technews.scheduler;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@ActiveProfiles("local")
@SpringBootTest
class ReleasesJavaSchedulerTest {
    @Autowired
    private ReleasesJavaScheduler releasesJavaScheduler;

    @Test
    void run() {
        releasesJavaScheduler.runScheduler();
    }
}

File: src/test/java/com/technews/scheduler/ReleasesJavaSchedulerTest.kt
package com.technews.scheduler

import com.technews.IntegrationTest
import io.kotest.core.spec.style.StringSpec
import org.junit.jupiter.api.Disabled
import org.springframework.beans.factory.annotation.Autowired

@Disabled
@IntegrationTest
class ReleasesJavaSchedulerTest(
    @Autowired private val releasesJavaScheduler: ReleasesJavaScheduler
) : StringSpec({
    "scrape a java release post" {
        releasesJavaScheduler.runScheduler()
    }
})
